sympy/physics/control/lti.py

```diff
@@ -2553,9 +2553,9 @@ def doit(self, cancel=True, expand=False, **hints):
         canceling the common factors then the ``cancel`` kwarg should be passed ``False``.
 
         >>> pprint(F_1.doit(cancel=False), use_unicode=False)
-        [           25*s*(1 - s)                          25 - 25*s              ]
-        [       --------------------                    --------------           ]
-        [       25*(1 - 6*s)*(1 - s)                    25*s*(1 - 6*s)           ]
+        [             s*(s - 1)                              s - 1               ]
+        [         -----------------                       -----------            ]
+        [         (1 - s)*(6*s - 1)                       s*(6*s - 1)            ]
```

Comment:
The reason for this change is that the inverse of this matrix has changed form:
```python
In [2]: M = Matrix([[-5*x/(1 - x) + 1, 0], [-25, 1]])

In [3]: M
Out[3]: 
⎡   5⋅x        ⎤
⎢- ───── + 1  0⎥
⎢  1 - x       ⎥
⎢              ⎥
⎣    -25      1⎦

In [4]: M.inv(method='DM') # new behaviour
Out[4]: 
⎡  x - 1     ⎤
⎢ ───────   0⎥
⎢ 6⋅x - 1    ⎥
⎢            ⎥
⎢25⋅x - 25   ⎥
⎢─────────  1⎥
⎣ 6⋅x - 1    ⎦

In [5]: M.inv(method='GE') # old behaviour
Out[5]: 
⎡     25        ⎤
⎢────────────  0⎥
⎢  125⋅x        ⎥
⎢- ───── + 25   ⎥
⎢  1 - x        ⎥
⎢               ⎥
⎢ 25⋅x - 25     ⎥
⎢ ─────────    1⎥
⎣  6⋅x - 1      ⎦
```
Both answers are correct. I guess the GE code only tries to cancel or canonicalise when it is looking for pivots or something.